### DISCUSSION TITLE
installare Home Assistant Supervised su docker su Mini PC Windows

### DISCUSSION THREAD

**Discussion by Emanuele-DV** [indomus/forum#1690] [HUB: Home Assistant]
Buonasera a tutti

sono nuovo è sicuramente inesperto, ho seguito e attivato da questo link Home Assistant, su un mini PC 

https://indomus.it/guide/come-installare-e-configurare-home-assistant-core-con-docker-su-windows/

Home Assistant Core con Docker su Windows

solo che una volta attivato mi sono che non aveva Supervised  e che Core era una versione per esperti 

ora vorrei disinstallare Core e nello stesso Docker installare la versione Home Assistant Supervised

Tutte le guide che trovo online, parlano di  raspberry

posso avere aiuto su come fare questo primo Passo ??? 

Grazie

**Comment by snakuzzo** [comment]
Ciao.
Installare "nello stesso Docker" non vuol dire assolutamente nulla.
L'installazione supervised non consiste in un unico container...bensì N container.
Innanzi tutto ti consiglierei di approfondire meglio il discorso docker.
Per quanto riguarda HA Supervised (su Docker), come abbiamo anche scritto nei nostri articoli, l'installazione è prevista solo su Linux ed è ufficialmente supportata solo su Debian (no derivate, anche se funzionerebbe uguale).
Se vuoi il supervisord le soluzioni sono due:
- installi linux e fai l'installazione supervised come da guida
- crei una VM ed importi l'immagine di hassos (disponibile sul sito ufficiale)

**Comment by KC-inDomus** [comment]
> Buonasera a tutti
> 
> 
> 
> sono nuovo è sicuramente inesperto, ho seguito e attivato da questo link Home Assistant, su un mini PC 
> 
> 
> 
> https://indomus.it/guide/come-installare-e-configurare-home-assistant-core-con-docker-su-windows/
> 
> 
> 
> Home Assistant Core con Docker su Windows
> 
> 
> 
> solo che una volta attivato mi sono che non aveva Supervised  e che Core era una versione per esperti 
> 
> 
> 
> ora vorrei disinstallare Core e nello stesso Docker installare la versione Home Assistant Supervised
> 
> 
> 
> Tutte le guide che trovo online, parlano di  raspberry
> 
> 
> 
> posso avere aiuto su come fare questo primo Passo ??? 
> 
> 
> 
> Grazie 

in sintesi: leggi [qui](https://indomus.it/formazione/home-assistant-su-mini-pc-intel-nuc-le-varie-installazioni-le-cose-da-fare/) e cerca di orientarti su cosa fare. altrimenti procedi a tentoni.
